gradle-plugins/model-compiler/src/main/java/io/spine/gradle/compiler/Extension.java

```diff
@@ -41,6 +46,7 @@
 // as this is a Gradle extension.
 public class Extension {
 
+    public static final String SPINE_BUILD_ARTIFACT_STORAGE_DIR = ".spine";
```

Comment:
Let's add a comment here, explaining what it is.

And an empty line between `public static final ... ;` block end and `private static final ...;` block start.